infrastructure/setup-deploy-config.sh

```diff
@@ -26,6 +26,19 @@ done
 KIBANA_ENCRYPTION_KEY=`uuidgen`
 sed -i "s/{{KIBANA_ENCRYPTION_KEY}}/$KIBANA_ENCRYPTION_KEY/g" /opt/opencrvs/infrastructure/monitoring/kibana/kibana.yml
 
+# Move metabase file
+mv /opt/opencrvs/infrastructure/metabase.init.db.sql /data/metabase/metabase.init.db.sql
```

Comment:
The metabase init SQL file is not read from the /data directory. This is because it doesn't ever change at runtime. There's also a security implication if we copy the file under `/data`: done this way, the deployment user needs write and read access to the /data directory which I'd really want to avoid.

https://github.com/opencrvs/opencrvs-farajaland/blob/develop/infrastructure/docker-compose.deploy.yml#L1019-L1020